pkg/shardddl/optimism/column.go

```diff
@@ -22,8 +24,9 @@ import (
 
 // GetAllDroppedColumns gets the all partially dropped columns.
 // return lockID -> column-name -> source-id -> upstream-schema-name -> upstream-table-name
```

Comment:
Please note what does `DropColumnStage` mean in comments.

Reply:
comments in https://github.com/pingcap/dm/blob/5ae19df6c531537f61d8b961bbbf526fe09fab80/pkg/shardddl/optimism/lock.go#L31-L40